woocommerce/wc-admin
# Add the Mollie payment provider setup task

Fixes #6042

Changes:

* Add the Mollie payment provider screen to the payment setup tasks
* Fix a bug where Klarna was not shown for "United Kingdom" (the country code is GB not UK)
* Whitelist the mollie plugin for install
* Require `wp-admin/file.php` when installing to avoid a fatal exception in the mollie activation hook.
* Fix a bug where an `Object` was passed to the thrown error in the plugin install action in `@woocommerce/data`

### Screenshots

<img width="724" alt="Screenshot 2021-02-04 at 2 38 37 PM" src="https://user-images.githubusercontent.com/1281828/106832568-c6e7cc80-66f6-11eb-8a37-3316f4abfeac.png">
<img width="740" alt="Screenshot 2021-02-04 at 2 38 43 PM" src="https://user-images.githubusercontent.com/1281828/106832577-ca7b5380-66f6-11eb-9d40-0cea394fdcf7.png">
<img width="730" alt="Screenshot 2021-02-04 at 2 38 57 PM" src="https://user-images.githubusercontent.com/1281828/106832580-cb13ea00-66f6-11eb-9105-c1330e77d661.png">

### Detailed test instructions:

1. You'll need a site that has the setup task list visible. Complete the OBW and make sure you're in a Mollie supported country (Such as United Kingdom).
2. Go to the setup payments task
3. Mollie should be listed as an option
4. Click "Set up"  button on the Mollie task
5. It should install and activate the mollie payments plugin
6. The connect step should provide links to create an account or go straight to Mollie settings. (test both links work)
7. Click "continue"
8. You should arrive back at the payment provider list

In step 6 the Mollie settings link wouldn't work if the plugin hadn't installed and activated but just check and confirm in the plugins list that Mollie was installed and activated.
